cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -45,7 +45,8 @@ spec:
   mgr:
     modules:
     # Several modules should not need to be included in this list. The "dashboard" and "monitoring" modules
-    # are already enabled by other settings in the cluster CR and the "rook" module is always enabled.
+    # are already enabled by other settings in the cluster CR. "rook" module is not enabled by default.
```

Comment:
If not enabled then I'm not sure if it's worth mentioning at all. If we do this for the "rook" module we might do it for all the modules... So I'd just remove it from comment entirely.

Reply:
Earlier it was enabled by default, imo we should at least notify about the change. I have removed the entire comment for now.